src/main/java/org/terasology/navgraph/Floor.java

```diff
@@ -61,20 +83,40 @@ public void merge(Region neighbor) {
         }
     }
 
+    /**
+     * Resets Entrances
```

Comment:
Like my above comment, i can tell it'll reset entrances from the name of the method, but what does resetting an entrance involve? Does it reset each entrance to some state? Does it delete them? 

A better description here would be "Removes all entrances from the floor, and sets them to null"

Still short and easy to write, it just tells me more than i would get from the name